microbench/src/main/java/alluxio/inode/InodeBenchWrite.java

```diff
@@ -43,38 +45,30 @@
 public class InodeBenchWrite {
 
   @State(Scope.Thread)
-  public static class ThreadState {
+  public static class ThreadState extends BaseThreadState {
```

Comment:
Can you throw exception if mBaseFileCount is not 0 since that param is not used here?